view/adminhtml/web/vue/field-type/checkbox.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+    <div class="admin__field field field-title">
+        <label
+            class="label admin__field-label"
+            :for="fieldId"
+        >
+            {{ label }}
+        </label>
+        <div class="admin__field-control control">
+            <input
+                class="input-text admin__control-text"
+                :id="fieldId"
+                type="checkbox"
+                v-model.number="checkboxValue"
+            />
+        </div>
+    </div>
+</template>
+<script>
+define(["Vue"], function(Vue) {
+    Vue.component("checkbox", {
+        template: template,
+        props: ['label', 'id', 'value', 'item'],
+        data: function() {
+            return {
+                fieldId: '',
+                checkboxValue: this.value
+            }
+        },
+        mounted: function() {
+            this.fieldId = 'snowmenu_' + this.id + '_' + this._uid;
+        },
+        watch: {
+          checkboxValue: function(newValue, oldValue) {
+            this.item.is_active = newValue ? '1' : '0'
```

Comment:
We shouldn't mutate the prop directly. If it's just an UI thing move it to computed.

Reply:
Yes. I agree. But we would first to refactor few things related to saving whole menu data. and I think it should be done in a new ticket